examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,91 @@
+# Usage
```

Comment:
Should these letters examples be part of this PR? The title says this is just checks.